### DISCUSSION TITLE
Can the server column be upright by default?

### DISCUSSION THREAD

**Discussion by GitHubkaede** [Eight-P/BetterDiscord-Stuff#18] [Q&A] [answered]
If there is a value & switch that can be set, I hope you can tell me.

 (horizontal server column is also nice)

**Comment by Eight-P** [comment, marked as answer]
No, unlikely, I see the server bar as a key feature of T1 and also it would require quite a bit of work.

**Reply by Uni404x64** [reply]
I'm rather disappointed in this actually this was a really **** response it's entirely possible and easily implemented
